week3/possible-qa-exercises/2-react-testing-library-exercises/src/5-Context/my-profile.jsx

```diff
@@ -0,0 +1,41 @@
+import {useContext} from "react";
+
+import LoggedInUserContext from "./logged-in-user-context";
+
+import TEST_ID from "./testids";
+
+function MyProfile() {
+  const {loggedInUser, isLoggedIn, logout} = useContext(LoggedInUserContext);
+
+  if (isLoggedIn) {
+    return (
+        <div>
+          <p>{`Welcome back ${loggedInUser.name}!`}</p>
+          {/* Here may be some information of the user, maybe some notifications.
+        Cut out for simplicity */}
```

Comment:
```suggestion
          {/* Here may be some information of the user, maybe some notifications. Cut out for simplicity */}
```

Reply:
I've left the break at 80 chars but I added spacing to improve readability